src/benchmark_ocl.cpp

```diff
@@ -41,42 +41,17 @@ using namespace ham::util;
 	#define WARP_SIZE 32
 #endif
 
-bool ocl_error_handler(cl_int err, const std::string& function_name, bool terminate = true)
+// integrate into benchmark()?
```

Comment:
Yes, it's the only call-site, so let's just remove that function and put its body directly into the benchmark lambda.

Reply:
Done with a758cba0fa201d64c477ff298ffa7ea784117a09.